graphql/tests/query.rs

```diff
@@ -154,6 +163,7 @@ fn test_schema(id: DeploymentHash, id_type: IdType) -> Schema {
         id: ID!
         name: String!
         members: [Musician!]! @derivedFrom(field: \"bands\")
+        reviews: [BandReview] @derivedFrom(field: \"band\")
```

Comment:
Is there a reason why this can't be `[BandReview!]!` like the other fields? IIRC, we do not allow arrays with nulls in them anyway. (There's a couple more places here that raise the same question)